config/settings/local.py

```diff
@@ -104,3 +104,7 @@
 AUDIT_LOG_ENABLED = True
 
 DISABLE_RATELIMIT = True
+
+# Uncomment when setting up on mac
+# GDAL_LIBRARY_PATH = '/opt/homebrew/opt/gdal/lib/libgdal.dylib'
+# GEOS_LIBRARY_PATH = '/opt/homebrew/opt/geos/lib/libgeos_c.dylib'
```

Comment:
i think adding a flag like GDAL_MANUAL_PATH would be better rather than commenting and uncommenting...

```suggestion
GDAL_MANUAL_PATH = env.bool("GDAL_MANUAL_PATH", default=False)
if GDAL_MANUAL_PATH:
    GDAL_LIBRARY_PATH = env("GDAL_LIBRARY_PATH", default=None)
    GEOS_LIBRARY_PATH = env("GEOS_LIBRARY_PATH", default=None)
```

Reply:
Yeah, this is a better implementation. Let's go with this.